populateStds.js

```diff
@@ -19,40 +19,59 @@ const legend =`
 - Entries in the effective column marked with an * are stable right now but turn to effective documents in the near future
 - Entries in the effective column marked with a † will turn deprecated in the near future`;
 
+// this one component will be copied to all files containing tables
+// the overview.mdx will always only be tableId 1, the other track files tableId 2
+// even though we have some code in the files that is not needed, it reduces read complexity in this file
 const reactHighlightTableCellBackground = `
 import { useEffect } from 'react';
+
 export const TableCellStyleApplier = () => {
     // apply background color based on cell index
-    const colorMapping = {
-      3: '#FBFDE2', // draft
-      4: '#E2EAFD', // effective
-      5: '#FDE2E2' // deprecated
+    const colorMappingPerTable = {
```

Comment:
Please make two consts, such as `colorMappingOverview` and `colorMappingTrackOverview`.

Reply:
Agreed, done.